lib/routes/routeWebsite.js

```diff
@@ -0,0 +1,67 @@
+import { errors } from 'arsenal';
+
+import api from '../api/api';
+import pushMetrics from '../utilities/pushMetrics';
+import routesUtils from './routesUtils';
+import statsReport500 from '../utilities/statsReport500';
+
+export default function routerWebsite(request, response, log, utapi,
+    statsClient) {
+    log.debug('routing request', { method: 'routerWebsite' });
+    // website endpoint only supports GET and HEAD and must have a bucket
+    // http://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteEndpoints.html
+    if ((request.method !== 'GET' && request.method !== 'HEAD')
+        || !request.bucketName) {
```

Comment:
@nicolas2bert may be able to confirm, but I believe if there is no bucket specified in the request, AWS behavior is to redirect to aws s3 homepage, not to return MethodNotAllowed. Is it a deliberate choice for us to return MethodNotAllowed instead?